modules/light-clients/07-tendermint/upgrade_test.go

```diff
@@ -238,13 +283,11 @@ func (suite *TendermintTestSuite) TestVerifyUpgrade() {
 
 				upgradedClientProof = []byte("proof")
 			},
-			expPass: false,
+			expErr: fmt.Errorf("could not unmarshal client merkle proof"),
```

Comment:
someone will likely PR a change here to use `errors.New()` since there no formatted args, I recall a recent PR which changed these types of error instantiations